src/probnum/filtsmooth/statespace/sde.py

```diff
@@ -105,85 +128,84 @@ def transition_rv(self, rv, start, stop, step, **kwargs):
             start,
             stop,
             step,
-            self._driftfun,
-            self._driftmatrixfun,
-            self._dispmatrixfun,
+            self.driftfun,
+            self.driftmatfun,
+            self.dispmatfun,
         )
 
     @property
     def dimension(self):
         """
         Spatial dimension (utility attribute).
         """
-        return len(self._driftmatrixfun(0.0))
+        # risky to evaluate at zero, but usually works
+        return len(self.driftmatfun(0.0))
 
 
 class LTISDE(LinearSDE):
     """
     Linear time-invariant continuous Markov models of the
     form
-    dx = [F x(t) + u] dt + L dBt.
+
+    .. math:: d x_t = [G x_t + v] d t + L d w_t.
```

Comment:
I think I prefered the notation with functions instead of indices. But feel free to just resolve this and leave things as they are!

Reply:
I'll consider this, and compare to the remaining code in sde.py